Source/Scene/ImplicitSubtree.js

```diff
@@ -2,18 +2,28 @@ import defaultValue from "../Core/defaultValue.js";
 import DeveloperError from "../Core/DeveloperError.js";
 import defined from "../Core/defined.js";
 import getJsonFromTypedArray from "../Core/getJsonFromTypedArray.js";
-import when from "../ThirdParty/when.js";
+import BufferCacheResource from "./BufferCacheResource.js";
 import has3DTilesExtension from "./has3DTilesExtension.js";
 import ImplicitAvailabilityBitstream from "./ImplicitAvailabilityBitstream.js";
 import ImplicitSubdivisionScheme from "./ImplicitSubdivisionScheme.js";
+import MetadataTable from "./MetadataTable.js";
+import ResourceCache from "./ResourceCache.js";
+import ResourceCacheKey from "./ResourceCacheKey.js";
+import when from "../ThirdParty/when.js";
 
 /**
  * An object representing a single subtree in an implicit tileset
  * including availability.
+ * <p>
+ * Subtrees handle tile metadata from the <code>3DTILES_metadata</code> extension
+ * </p>
+ *
+ * @see {@link https://github.com/CesiumGS/3d-tiles/tree/3d-tiles-next/extensions/3DTILES_metadata/1.0.0#implicit-tile-metadata|Implicit Tile Metadata in the 3DTILES_metadata specification}
  *
  * @alias ImplicitSubtree
  * @constructor
  *
+ * @param {Cesium3DTileset} tileset The tileset this subtree belongs to. Used for accessing metadata properties
```

Comment:
Is `tileset` used here anymore?

Reply:
ah no, I had moved that to `ImplicitTileset`, forgot to move the comment.